File: length_last_word.py
class length_last_word:
    def lengthOfLastWord(self, s: str) -> int:
        return len(s.split()[-1])

        # alternate approach
        # l = []
        # count = 0
        # for char in s:
        #     if char != ' ':
        #         count += 1
        #     else:
        #         if count != 0:
        #             l.append(count)
        #             count = 0
        # # handle last word count
        # if count != 0:
        #     l.append(count)
        # return l[-1] if l else 0

File: length_last_word.java
public class length_last_word {
    public int lengthOfLastWord(String s) {
        s = s.trim();
        // s = s + " ";
        int wordlen = 0;
        for (int i = s.length()-1; i >= 0; i--) {
            char ch = s.charAt(i);
            if (ch != ' ') {
                wordlen++;
            } else if (wordlen > 0) {
                break;
            }
        }
        return wordlen;
    }
}
